package.json

```
@@ -32,18 +32,18 @@
     "package-tarball": "npm run build && npm pack"
   },
   "devDependencies": {
-    "@sketch-hq/sketch-assistant-utils": "6.0.0",
-    "@types/jest": "26.0.4",
-    "@types/node": "14.0.22",
-    "jest": "26.6.3",
+    "@sketch-hq/sketch-assistant-utils": "6.4.0",
```

Small change required given the latest release:

```suggestion
    "@sketch-hq/sketch-assistant-utils": "6.4.1",
```